code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -160,9 +160,7 @@
 /datum/symptom/heal/metabolism/Heal(mob/living/carbon/C, datum/disease/advance/A, actual_power)
 	if(!istype(C))
 		return
-	C.reagents.metabolize(C, can_overdose=TRUE) //this works even without a liver; it's intentional since the virus is metabolizing by itself
-	if(triple_metabolism)
-		C.reagents.metabolize(C, can_overdose=TRUE)
+	C.reagents.metabolize(C, triple_metabolism ? 4 : 2, 0, can_overdose=TRUE) //this works even without a liver; it's intentional since the virus is metabolizing by itself
```

Comment:
I do not like this

Reply:
Looking at it again I don't either.

Reply:
It's just about the best I can do without passing through delta_time and symptoms have their own time handling system.